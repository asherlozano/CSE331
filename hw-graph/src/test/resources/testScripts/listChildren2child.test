# listChildren2Child expected file
# tests if the listChildren method can be added to graph
# listChildren lists all the children of n1

# Create a graph
CreateGraph  g1

# Create a node
AddNode g1 n1
AddNode g1 n2
AddNode g1 n3

# Create an edge
AddEdge g1 n1 n2 e12
AddEdge g1 n2 n3 e23

# Print the nodes in the graph
ListNodes g1
ListChildren g1 n2