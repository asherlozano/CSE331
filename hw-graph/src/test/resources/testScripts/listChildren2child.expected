# listChildren2Child expected file
# tests if the listChildren method can be added to graph
# listChildren lists all the children of n1

# Create a graph
created graph g1

# Create a node
added node n1 to g1
added node n2 to g1
added node n3 to g1

# Create an edge
added edge e12 from n1 to n2 in g1
added edge e23 from n2 to n3 in g1

# Print the nodes in the graph
g1 contains: n1 n2 n3
the children of n1 in g1 are: n2(e12) n3(e23)